install_config/install/prerequisites.adoc

```diff
@@ -797,6 +797,45 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[managing-docker-container-logs]]
+*Managing Docker Container Logs*
+
+If your container's log (*_/var/lib/docker/containers/HASH/HASH-json.log_*) is
+increasing, you can use `json-file` logging driver options to manage this.
+
```

Comment:
Could the problematic issue be stressed somehow? Aren't log files always increasing? What's the danger here? What happens if you don't configure the max-size and max-file numbers for the docker container logs? 

Maybe something like: "Sometimes a container's log file (*_/var/lib/docker/containers/HASH/HASH-json.log_*) can increase to a problematic size. You can manage this by configuring the `json-file' logging driver to restrict the size and number of log files."